packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -118,11 +119,11 @@ export async function generateImageData({
     reporter.warn(
       `Specifying fullWidth images will ignore the width and height arguments, you may want a constrained image instead. Otherwise, use the breakpoints argument.`
     )
-    args.width = metadata.width
+    args.width = metadata?.width
```

Comment:
Is there a scenario where metadata will be undefined entirely? It looks like in the worst case it's an empty object.

Reply:
If sharp throws an error getting metadata it would be undefined

Reply:
I'll change it so it's an empty object though